### PR TITLE
Don't install apparmor (and uninstall it) on Ubuntu 16.04+

### PR DESCRIPTION
**Pull request by tas50**

### CODE REVIEW

**Comment by tas50** [comment]
Fixed @eheydrick's issue

File: recipes/default.rb [hunk 1]
```diff
@@ -28,6 +28,11 @@
     package ntppkg
   end
 
+  package 'ntpdate' do
+    action :remove
+    only_if { node['platform_version'].to_i >= 16 }
```
Comment (shortdudey123) [inline recipes/default.rb]: If i am reading it right, this will affect Fedora as well, not just Ubuntu

Comment (tas50) [inline recipes/default.rb]: Oh yes. I missed that bigtime. I fixed it with a platform_family requirement

File: recipes/default.rb [hunk 2]
```diff
@@ -28,6 +28,11 @@
     package ntppkg
   end
 
+  package 'ntpdate' do
+    action :remove
+    only_if { node['platform_version'] == 'debian' && node['platform_version'].to_i >= 16 }
```
Comment (shortdudey123) [inline recipes/default.rb]: @tas50 `platform_version`? you mean `platform_family`? 😄

Comment (shortdudey123) [inline recipes/default.rb]: Maybe `node['platform'] == 'ubuntu'` would be better though

**Pull request merged**